lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+require 'pry'
+
+class Human < Player
+
+  def move(board)
+    puts "Please enter 1-9:"
+    position = gets.strip
+  end  
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  puts "Please enter 1-9:"
  gets.strip
end
```